mlir/include/mlir/Tools/lsp-server-support/SourceMgrUtils.h

```diff
@@ -14,12 +14,14 @@
 #ifndef MLIR_TOOLS_LSPSERVERSUPPORT_SOURCEMGRUTILS_H
 #define MLIR_TOOLS_LSPSERVERSUPPORT_SOURCEMGRUTILS_H
 
-#include "mlir/Tools/lsp-server-support/Protocol.h"
+#include "mlir/Support/LLVM.h"
+#include "llvm/Support/LSP/Protocol.h"
 #include "llvm/Support/SourceMgr.h"
 #include <optional>
 
 namespace mlir {
 namespace lsp {
+using namespace llvm::lsp;
```

Comment:
That seems risky: any change in LLVM could create a conflict with a name in MLIR.

Can you the actual name you need instead of the entire namespace?

Like: `using llvm::lsp::ErrorCode;` for example?

Reply:
+1